src/net/sf/memoranda/ui/ImportSticker.java

```diff
@@ -1,27 +1,83 @@
 package net.sf.memoranda.ui;
 
+
+import javax.swing.SwingUtilities;
+import javax.swing.JEditorPane;
 import javax.swing.JOptionPane;
+import javax.swing.JFileChooser;
+import javax.swing.JScrollPane;
 
 import net.sf.memoranda.util.Local;
+import java.io.*;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import net.sf.memoranda.CurrentProject;
+import net.sf.memoranda.EventNotificationListener;
+import net.sf.memoranda.EventsManager;
+import net.sf.memoranda.EventsScheduler;
+import net.sf.memoranda.History;
+import net.sf.memoranda.NoteList;
+import net.sf.memoranda.Project;
+import net.sf.memoranda.ProjectListener;
+import net.sf.memoranda.ProjectManager;
+import net.sf.memoranda.ResourcesList;
+import net.sf.memoranda.TaskList;
+import net.sf.memoranda.date.CalendarDate;
+import net.sf.memoranda.date.CurrentDate;
+import net.sf.memoranda.date.DateListener;
+import net.sf.memoranda.util.AgendaGenerator;
+import net.sf.memoranda.util.CurrentStorage;
+import net.sf.memoranda.util.Local;
+import net.sf.memoranda.util.Util;
+import nu.xom.Element;
+import javax.swing.event.HyperlinkEvent;
+import javax.swing.event.HyperlinkListener;
 
+/**ImportStricker class 
+ * 
+ * Finds file and reads file then creates stickies
+ */
 public class ImportSticker {
 
-String name;        
-        
-        public ImportSticker(String x) {
-                name = x;
-        }
-
-        public boolean import_file(){
-                /*
-                 We are working on this =)
-                  
-                  
-                  */
+	 private String name;        
+	 private JEditorPane viewer = new JEditorPane("text/html", "");
+	
+	public ImportSticker(String x) {
+		name = x;
+	}
+	
+	/**Import Sticker file creates stickies 
+	 * 
+	 * @return boolean
+	 */
+    public boolean import_file(){
```

Comment:
Though you didn't create this method, since you implemented it, it would probably be best to rename the method to match the coding standards with valid camelCase and no underscores except as a prefix to private members/methods (CG: 4.d)